jupyter/jupyter-web-app/base/configs/spawner_ui_config.yaml

```diff
@@ -18,7 +18,11 @@ spawnerFormDefaults:
   image:
     # The container Image for the user's Jupyter Notebook
     # If readonly, this value must be a member of the list below
+<<<<<<< HEAD
```

Comment:
I think we should go with the HEAD changes :)
Also the generated test data has these lines, I guess you'll have to regenerate the tests?

Reply:
Good catch thanks.